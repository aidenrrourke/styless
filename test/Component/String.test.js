import React from 'react';
import renderer from 'react-test-renderer';
import styled from 'styled-components';
import 'jest-styled-components';

test('String characters @url', () => {
    const Div = styled.div`
      background-image: url('images/lamp-post.png?v=1');
	`;
    expect(renderer.create(<Div/>).toJSON()).toMatchSnapshot();
});