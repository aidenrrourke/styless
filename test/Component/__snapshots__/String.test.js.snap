// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`String characters @url 1`] = `
.c0 {
  background-image: url('images/lamp-post.png?v=1');
}

<div
  className="c0"
/>
`;
